serverless.yml

```diff
@@ -28,7 +28,11 @@ provider:
     NETLIFY_TOKEN: ${env:NETLIFY_TOKEN}
     YELP_API_KEY: ${env:YELP_API_KEY}
     SLACK_OAUTH: ${env:SLACK_OAUTH}
-
+    GOOGLE_CLIENT_ID: ${env:GOOGLE_CLIENT_ID}
```

Comment:
for smaller projects, or environments that we have well control, one way to be lazy about this is:

```yaml
environment: ${env:}
```

where all env vars that the `serverless` process (the CLI) sees get relayed into its serverless/lambda configuration. _not_ necessarily bad if these env vars are well controlled.

but a `print-env` plugin for serverless is probbly still mroe ideal.

a middle-ground can be that env vars are sourced through portunus/print-env and written into a dot env file, and then utilize https://www.serverless.com/plugins/serverless-dotenv-plugin